tests/ml_commons/test_ml_commons_client.py

```diff
@@ -372,6 +372,26 @@ def test_integration_model_train_register_full_cycle():
                     print("Model Task Status:", ml_task_status)
                     raised = True
                 assert raised == False, "Raised Exception in pulling task info"
+
+                try:
+                    search_task_obj = ml_client.search_task(json="{'_id':'{task_id}'}")
+                    assert search_task_obj["hits"]["hits"][0]["_id"] == task_id
+                except:  # noqa: E722
+                    raised = True
+                assert raised == False, "Raised Exception in searching task"
+
+                try:
+                    search_task_obj = ml_client.search_task(json={"_id": task_id})
+                except:  # noqa: E722
+                    raised = True
+                assert raised == False, "Raised Exception in searching task"
+
+                try:
+                    search_task_obj = ml_client.search_task(json=15)
+                    assert search_task_obj == "Invalid JSON object passed as argument."
+                except:  # noqa: E722
+                    raised = True
+                assert raised == False, "Raised Exception in searching task"
```

Comment:
can we make a separate function for this to test? This chain of flow test was to test integration test. We don't need to include this test here.

Reply:
> can we make a separate function for this to test? This chain of flow test was to test integration test. We don't need to include this test here.

Ok.